automated install/basic-install.sh

```diff
@@ -165,11 +167,11 @@ if command -v apt-get &> /dev/null; then
     # Check for and determine version number (major and minor) of current php install
     if command -v php &> /dev/null; then
         phpInsVersion="$(php -v | head -n1 | grep -Po '(?<=PHP )[^ ]+')"
-        echo -e "  ${INFO} Existing PHP installation detected : PHP version $phpInsVersion"
-        phpInsMajor="$(echo "$phpInsVersion" | cut -d\. -f1)"
-        phpInsMinor="$(echo "$phpInsVersion" | cut -d\. -f2)"
+        printf "  %b Existing PHP installation detected : PHP version %s\\n" "${INFO}" "${phpInsVersion}"
+        printf -v phpInsMajor "%d" "${phpInsVersion%%.*}"
+        printf -v phpInsMinor "%d" "${phpInsVersion##*.}"
```

Comment:
`phpInsMinor` does not get parsed correctly (the original code took the second match, and this takes from the end):
```
$ php -v | head -n1 | grep -Po '(?<=PHP )[^ ]+'
7.0.32-0ubuntu0.16.04.1
$ phpInsVersion="$(php -v | head -n1 | grep -Po '(?<=PHP )[^ ]+')" && printf "%d" "${phpInsVersion%%.*}"
7
$ phpInsVersion="$(php -v | head -n1 | grep -Po '(?<=PHP )[^ ]+')" && printf "%d" "${phpInsVersion##*.}"
1
```

Reply:
I'm not sure what the minor revision is useful for. It's used to check for a version >= 7.0. I'll fix that grab to be less greedy, but is there a need to know what the minor version is?

Reply:
I don't think so, at least now. The most it does is print out alongside the major version.

Reply:
Thanks, I see where that comes in now. Digital Ocean's Ubuntu 18.10 is giving me problems right now with their apt mirror missing a Release file so once that is fixed I'll get something working here.

Reply:
What do you think of something like `echo "<?php echo PHP_MAJOR_VERSION ?>" | php`

Just asking PHP directly what major and minor version is installed?

Reply:
That sounds like a much better way :+1: